tests/unit_tests_old/tests_launch/test_launch_cli.py

```diff
@@ -68,11 +67,9 @@ def test_agent_queues_notfound(runner, test_settings, live_mock_server):
         result = runner.invoke(
             cli.launch_agent,
             [
-                "--project",
-                "test_project",
                 "--entity",
                 "mock_server_entity",
-                "--queues",
+                "--queue",
```

Comment:
Could you add a `queues` key to the config file in the test `test_launch_agent_base` in this file and validate that it will use the specified queues from the config file?